R/community_overlap.R

```diff
@@ -1,8 +1,10 @@
-#' Community Overlap Calculation
+#' Community overlap Calculation
```

Comment:
If overlap isn't capitalized, calculation should also not be

Reply:
fixed